### PR TITLE
Added Free Mobile connector

### PR DESCRIPTION
**Pull request by ZeHiro**
Known problem : 
If bad credential are used to try the import, and fail, if credentials are then corrected, it is impossible to import without restarting Konnectors (or maybe wait)

For update of ware, see commit comment (not tested with all konnectors, only free and freemobile)

### CODE REVIEW

File: server/konnectors/freemobile.coffee [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+    init: (callback) ->
+        map = (doc) -> emit doc.date, doc
+        PhoneBill.defineRequest 'byDate', map, (err) ->
+            callback err
+
+    fetch: (requiredFields, callback) ->
+
+        log.info "Import started"
+
+        fetcher.new()
```
Comment (frankrousseau) [inline server/konnectors/freemobile.coffee]: It's great that you used the Fetcher. Thank you for taking the time to understand how it works. That's hard without documentation.

**Comment by frankrousseau** [comment]
I merged this PR because you did an awesome work. 

But next time, please follow the [Cozy code guidelines](https://github.com/cozy/cozy-guidelines).

**Comment by frankrousseau** [comment]
Thank you a lot for this Konnector!

**Comment by ZeHiro** [comment]
Ok, i can re submit the PR with the 80 chars rules if you prefer, it won't take time.

**Pull request merged**